<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Driller</string>
  </property>
  <widget class="QWidget" name="verticalLayout_2" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>6</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Database name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="databaseName" />
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListView" name="tableList" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="horizontalScrollBarPolicy" >
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayout" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Data offset:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Table name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="tableName" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>File name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="tableFileName" />
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Row length:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QSpinBox" name="tableDataOffset" >
            <property name="maximum" >
             <number>1000000</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QSpinBox" name="tableRowLength" >
            <property name="maximum" >
             <number>1000000</number>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="columnList" >
          <property name="editTriggers" >
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::NoEditTriggers|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="uniformRowHeights" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="addTableButton" >
            <property name="text" >
             <string>Add Table</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeTableButton" >
            <property name="text" >
             <string>Remove Table</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addItemButton" >
            <property name="text" >
             <string>Add Column</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeItemButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Select an item</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDriller" >
    <property name="title" >
     <string>&amp;Driller</string>
    </property>
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuDatabase" >
    <property name="title" >
     <string>D&amp;atabase</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="actionExtract_Data" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRedo" />
    <addaction name="separator" />
    <addaction name="actionCut" />
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
    <addaction name="actionDelete" />
    <addaction name="actionSelect_All" />
    <addaction name="separator" />
    <addaction name="actionSettings" />
   </widget>
   <addaction name="menuDriller" />
   <addaction name="menuDatabase" />
   <addaction name="menuEdit" />
   <addaction name="menuHelp" />
  </widget>
  <action name="actionOpen" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="text" >
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExtract_Data" >
   <property name="text" >
    <string>&amp;Extract Data</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionUndo" >
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo" >
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="text" >
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDelete" >
   <property name="text" >
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_All" >
   <property name="text" >
    <string>&amp;Select All</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSettings" >
   <property name="text" >
    <string>S&amp;ettings</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
